src/main/java/lotto/model/Lottos.java

```diff
@@ -0,0 +1,46 @@
+package lotto.model;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Lottos {
+    private final List<Lotto> lottos;
+
+    public Lottos(int count) {
+        this(new ArrayList<>());
+        for (int i = 0; i < count; i++) {
+            lottos.add(new Lotto());
+        }
```

Comment:
```suggestion
        this(createLottos());
```
 생성자에서 로직 구현하지 말고 다른 메서드를 추가해 호출하는 방식으로 구현할 것을 추천